CI failing due to pkg_resources in upstream

See latest actions: https://github.com/openforcefield/openff-evaluator/actions/runs/21847035919

```
../../../micromamba/envs/evaluator-test-env/lib/python3.11/site-packages/smirnoff99frosst/smirnoff99frosst.py:11: in <module>
    from pkg_resources import resource_filename
E   ModuleNotFoundError: No module named 'pkg_resources'
```

Probably related to https://github.com/openforcefield/openff-nagl/issues/239

--- a/devtools/conda-envs/docs_env.yaml
+++ b/devtools/conda-envs/docs_env.yaml
@@ -4,7 +4,7 @@ channels:
 dependencies:
     # Base depends
     - python =3.11
-    - setuptools =81
+    - setuptools
     - pip
 
     # Standard dependencies

--- a/devtools/conda-envs/test_env.yaml
+++ b/devtools/conda-envs/test_env.yaml
@@ -15,7 +15,6 @@ dependencies:
   - nbval
   - requests-mock # For testing http requests.
   - foyer
-  - smirnoff99frosst
 
     # Shim
   - pint >=0.21,<0.25

--- a/devtools/conda-envs/test_env.yaml
+++ b/devtools/conda-envs/test_env.yaml
@@ -4,7 +4,6 @@ channels:
 dependencies:
     # Base depends
   - python
-  - setuptools =81
   - pip
 
     # Testing and development

--- a/openff/evaluator/_tests/test_client.py
+++ b/openff/evaluator/_tests/test_client.py
@@ -38,9 +38,7 @@ def test_default_options():
     """Test creating the default estimation options."""
 
     data_set = PhysicalPropertyDataSet()
-    force_field_source = SmirnoffForceFieldSource.from_path(
-        "openff-2.2.1.offxml"
-    )
+    force_field_source = SmirnoffForceFieldSource.from_path("openff-2.2.1.offxml")
 
     for property_type in property_types:
         physical_property = create_dummy_property(property_type)

--- a/openff/evaluator/_tests/test_client.py
+++ b/openff/evaluator/_tests/test_client.py
@@ -39,7 +39,7 @@ def test_default_options():
 
     data_set = PhysicalPropertyDataSet()
     force_field_source = SmirnoffForceFieldSource.from_path(
-        "smirnoff99Frosst-1.1.0.offxml"
+        "openff-2.2.1.offxml"
     )
 
     for property_type in property_types:
@@ -61,7 +61,7 @@ def test_default_options():
     "force_field_source, expected_protocol_type",
     [
         (
-            SmirnoffForceFieldSource.from_path("smirnoff99Frosst-1.1.0.offxml"),
+            SmirnoffForceFieldSource.from_path("openff-2.2.1.offxml"),
             "BuildSmirnoffSystem",
         ),
         (TLeapForceFieldSource(), "BuildTLeapSystem"),
@@ -97,7 +97,7 @@ def test_submission():
                     client = EvaluatorClient()
 
                     # Submit an empty data set.
-                    force_field_path = "smirnoff99Frosst-1.1.0.offxml"
+                    force_field_path = "openff-2.2.1.offxml"
                     force_field_source = SmirnoffForceFieldSource.from_path(
                         force_field_path
                     )

--- a/openff/evaluator/_tests/test_layers/test_reweighting.py
+++ b/openff/evaluator/_tests/test_layers/test_reweighting.py
@@ -93,7 +93,7 @@ def test_storage_retrieval():
         },
     }
 
-    force_field = SmirnoffForceFieldSource.from_path("smirnoff99Frosst-1.1.0.offxml")
+    force_field = SmirnoffForceFieldSource.from_path("openff-2.2.1.offxml")
 
     with tempfile.TemporaryDirectory() as base_directory:
         # Create a storage backend with some dummy data.

--- a/openff/evaluator/_tests/test_layers/test_workflow_layer.py
+++ b/openff/evaluator/_tests/test_layers/test_workflow_layer.py
@@ -56,7 +56,7 @@ def test_workflow_layer():
 
             # Set-up a simple storage backend and add a force field to it.
             force_field = SmirnoffForceFieldSource.from_path(
-                "smirnoff99Frosst-1.1.0.offxml"
+                "openff-2.2.1.offxml"
             )
 
             storage_backend = LocalFileStorage()

--- a/openff/evaluator/_tests/test_layers/test_workflow_layer.py
+++ b/openff/evaluator/_tests/test_layers/test_workflow_layer.py
@@ -55,9 +55,7 @@ def test_workflow_layer():
             os.makedirs(layer_directory)
 
             # Set-up a simple storage backend and add a force field to it.
-            force_field = SmirnoffForceFieldSource.from_path(
-                "openff-2.2.1.offxml"
-            )
+            force_field = SmirnoffForceFieldSource.from_path("openff-2.2.1.offxml")
 
             storage_backend = LocalFileStorage()
             batch.force_field_id = storage_backend.store_force_field(force_field)

--- a/openff/evaluator/_tests/test_storage/test_storage.py
+++ b/openff/evaluator/_tests/test_storage/test_storage.py
@@ -49,9 +49,7 @@ def test_force_field_storage():
     """A simple test to that force fields can be stored and
     retrieved using the local storage backend."""
 
-    force_field_source = SmirnoffForceFieldSource.from_path(
-        "openff-2.2.1.offxml"
-    )
+    force_field_source = SmirnoffForceFieldSource.from_path("openff-2.2.1.offxml")
 
     with tempfile.TemporaryDirectory() as temporary_directory:
         local_storage = LocalFileStorage(temporary_directory)

--- a/openff/evaluator/_tests/test_storage/test_storage.py
+++ b/openff/evaluator/_tests/test_storage/test_storage.py
@@ -50,7 +50,7 @@ def test_force_field_storage():
     retrieved using the local storage backend."""
 
     force_field_source = SmirnoffForceFieldSource.from_path(
-        "smirnoff99Frosst-1.1.0.offxml"
+        "openff-2.2.1.offxml"
     )
 
     with tempfile.TemporaryDirectory() as temporary_directory:

--- a/openff/evaluator/_tests/test_workflow/test_replicators.py
+++ b/openff/evaluator/_tests/test_workflow/test_replicators.py
@@ -51,7 +51,7 @@ def test_simple_replicators():
     dummy_property = create_dummy_property(Density)
 
     dummy_metadata = Workflow.generate_default_metadata(
-        dummy_property, "smirnoff99Frosst-1.1.0.offxml", []
+        dummy_property, "openff-2.2.1.offxml", []
     )
 
     dummy_workflow = Workflow(dummy_metadata, "")
@@ -125,7 +125,7 @@ def test_group_replicators():
     dummy_property = create_dummy_property(Density)
 
     dummy_metadata = Workflow.generate_default_metadata(
-        dummy_property, "smirnoff99Frosst-1.1.0.offxml", []
+        dummy_property, "openff-2.2.1.offxml", []
     )
 
     dummy_workflow = Workflow(dummy_metadata, "")
@@ -205,7 +205,7 @@ def test_advanced_group_replicators():
     dummy_property = create_dummy_property(Density)
 
     dummy_metadata = Workflow.generate_default_metadata(
-        dummy_property, "smirnoff99Frosst-1.1.0.offxml", []
+        dummy_property, "openff-2.2.1.offxml", []
     )
 
     dummy_workflow = Workflow(dummy_metadata, "")
@@ -263,7 +263,7 @@ def test_nested_replicators():
     dummy_property = create_dummy_property(Density)
 
     dummy_metadata = Workflow.generate_default_metadata(
-        dummy_property, "smirnoff99Frosst-1.1.0.offxml", []
+        dummy_property, "openff-2.2.1.offxml", []
     )
 
     dummy_workflow = Workflow(dummy_metadata, "")
@@ -311,7 +311,7 @@ def test_advanced_nested_replicators():
 
     dummy_property = create_dummy_property(Density)
     dummy_metadata = Workflow.generate_default_metadata(
-        dummy_property, "smirnoff99Frosst-1.1.0.offxml", []
+        dummy_property, "openff-2.2.1.offxml", []
     )
     dummy_metadata["dummy_list"] = [[1], [2]]
 

--- a/openff/evaluator/_tests/test_workflow/test_workflow.py
+++ b/openff/evaluator/_tests/test_workflow/test_workflow.py
@@ -313,7 +313,7 @@ def test_find_relevant_gradient_keys(tmp_path):
 
 def test_generate_default_metadata_defaults():
     dummy_property = create_dummy_property(Density)
-    dummy_forcefield = "smirnoff99Frosst-1.1.0.offxml"
+    dummy_forcefield = "openff-2.2.1.offxml"
 
     data = Workflow.generate_default_metadata(dummy_property, dummy_forcefield)
 

--- a/openff/evaluator/_tests/test_workflow/utils.py
+++ b/openff/evaluator/_tests/test_workflow/utils.py
@@ -10,7 +10,7 @@ from openff.evaluator.workflow.attributes import InputAttribute, OutputAttribute
 
 def create_dummy_metadata(dummy_property, calculation_layer):
     global_metadata = Workflow.generate_default_metadata(
-        dummy_property, "smirnoff99Frosst-1.1.0.offxml", []
+        dummy_property, "openff-2.2.1.offxml", []
     )
 
     if calculation_layer == "ReweightingLayer":

--- a/openff/evaluator/_tests/utils.py
+++ b/openff/evaluator/_tests/utils.py
@@ -228,12 +228,12 @@ def build_tip3p_smirnoff_force_field():
     Returns
     -------
     SmirnoffForceFieldSource
-        The force field containing both smirnoff99Frosst-1.1.0
+        The force field containing both openff-2.2.1
         and TIP3P parameters
     """
     from openff.toolkit.typing.engines.smirnoff import ForceField
 
-    smirnoff_force_field_path = "smirnoff99Frosst-1.1.0.offxml"
+    smirnoff_force_field_path = "openff-2.2.1.offxml"
     tip3p_force_field_path = get_data_filename("forcefield/tip3p.offxml")
 
     smirnoff_force_field_with_tip3p = ForceField(

--- a/openff/evaluator/client/client.py
+++ b/openff/evaluator/client/client.py
@@ -339,7 +339,7 @@ class EvaluatorClient:
     >>>
     >>> # Load in the force field parameters
     >>> from openff.evaluator.forcefield import SmirnoffForceFieldSource
-    >>> force_field_source = SmirnoffForceFieldSource.from_path('smirnoff99Frosst-1.1.0.offxml')
+    >>> force_field_source = SmirnoffForceFieldSource.from_path('openff-2.2.1.offxml')
     >>>
     >>> # Submit the estimation request to a running server.
     >>> request = client.request_estimate(data_set, force_field_source)

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -2,7 +2,7 @@
 build-backend = "setuptools.build_meta"
 
 requires = [
-  "setuptools==81",
+  "setuptools>=77.0.3",
   "setuptools-scm>=8",
 ]
 

